### ISSUE TITLE
Remove Kotlin runtime from rspeer.jar lib

### ISSUE THREAD

**Issue by purple-ice** [rs-peer/issues#383]
It's not supposed to be here and only breaks builds/imports due to Kotlin runtime version mismatch, because to extend Java with Kotlin, you configure Kotlin locally, and install your own runtime. Bundled runtime causes nothing but issues.

Secondly, rspeer is a Java project, not Kotlin project, all scripts written in Kotlin still are targeting JVM, in this case and runtime being used is Java, not Kotlin, therefore it's not even needed here in the first place. The only reason you need it on your end is on SDN, so you can support scripts written in Kotlin, **it's only required for compiling it, not running it**.

And most importantly, RSPeer does not depend on Kotlin runtime in any way.

Why is it here at all? It should be on SDN, in project that builds scripts, nowhere else.

As for my specific issue, it's pretty straightforward.

I am using latest version of IntelliJ IDEA with latest Kotlin plugin.

I create a plain Java project, set rspeer as provided library in settings, and then create a new Kotlin file. This is when IntelliJ tells me that Kotlin is not configured, so I configure it to use Kotlin plugin, like a normal person using IntelliJ would do.

And then, I get told that there's no such thing as `org.rspeer.*` and warnings clearly state that the issue is rspeer.jar, that contains bundled Kotlin runtime.

Current workaround that solves it:
```bash
ls ~/.rspeer/*.jar -t | head -n 1 | xargs -I F cp F ~/RSPeer/cache/rspeer.jar && zip -d ~/RSPeer/cache/rspeer.jar "kotlin/*" "META-INF/*.kotlin_module"
```

This one single line removes bundled runtime from the library and everything works again.

In other words
```
rspeer.jar
    kotlin/*
    META-INF/*.kotlin_module
```
Are the problem, which is not surprising.

Don't provide fat jars as libraries, thank you.

**Comment by 0xdeki** [comment]
+1, kotlin should be removed everywhere

**Comment by purple-ice** [comment]
That's fair, I personally am not planning to push Kotlin scripts to SDN since Spencer, who I talked to before getting told to just post it here, doesn't seem to care about it, and I know that it will break.

The only thing I'm asking is to stop breaking local builds while misunderstanding where Kotlin in this kind of project stands.

**Comment by MaddoxDevelopment** [comment]
It's not that Spencer does not care about your issue, it's the fact of how you approached it in PM's.

https://i.imgur.com/EJ6enWO.png

Does this seem like the correct way to approach a developer about an issue, berating them? Spencer didn't add Kotlin to the client, I did. I was told by a few people it was required so I went ahead and added it without a second thought. 

I suggest you work on your temper and whatever real life issues that are causing you to act this way. 

That being said, I'll look into removing it in the near future.

**Comment by purple-ice** [comment]
Speaking of professionalism, issues shouldn't be closed until they are actually resolved.



You are incapable of being professional about things and I don't see what's your problem with anything I said, especially when it's true.
